wasm-host/src/main.rs

```diff
@@ -137,7 +137,7 @@ fn main() {
 
     let data_source = MockData::new(&tx_json, &lo_json, &lh_json, &l_json, &nft_json);
     info!("Executing function: {}", args.function);
-    match vm_wamr::run_func(wasm_file, &args.function, Some(1000000), data_source) {
+    match vm_wamr::run_func(wasm_file, &args.function, Some(100000), data_source) {
```

Comment:
I reverted this in my PR, but it looks like `main` has been updated with the expanded value. I think this value should match whatever ripple is actually using.